src/site/content/en/blog/service-worker-mindset/index.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Service Worker Mindset
+subhead: How to think when thinking about Service Workers.
+authors:
+  - geddski
+date: 2019-05-28
+hero: hero.jpg
+hero_position: center
+alt: Service Workers lining up.
+description: |
+  Working with Service Workers is new and unfamiliar for many web devs. Here's some tips for wrapping your mind around them.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - service-worker
+  - PWA
+  - offline
+  - caching
+---
+
+Service Workers are powerful and absolutely worth mastering. They'll enable you to deliver an entirely new level of experience to your users. But Service Workers are unlike anything most of us web devs are used to. They come with a steep learning curve and a handful of snags you've got to watch out for.
+
+The best way to ensure success with Service Workers is to have the right mindset when working with them. In this post we'll wrap our brains around the paradoxical traits that make Service Workers both tricky and awesome.
+
+## The Same, but Different
```

Comment:
```suggestion
## The same, but different
```

Just need to update to sentence case for all headings.

Reply:
got i